src/overlays/actors/ovl_Bg_Spot06_Objects/z_bg_spot06_objects.c

```diff
@@ -45,13 +46,13 @@ void BgSpot06Objects_LockFloat(BgSpot06Objects* this, GlobalContext* globalCtx);
 void BgSpot06Objects_WaterPlaneCutsceneWait(BgSpot06Objects* this, GlobalContext* globalCtx);
 void BgSpot06Objects_WaterPlaneCutsceneRise(BgSpot06Objects* this, GlobalContext* globalCtx);
 
-extern Gfx D_06000120[];           // Lake Hylia lowered water
-extern Gfx D_06000470[];           // Lake Hylia raised water
-extern Gfx D_06000E10[];           // Water Temple entrance gate
-extern CollisionHeader D_06000EE8; // Water Temple entrance gate collision
-extern Gfx D_06001160[];           // Zora's Domain entrance block of ice
-extern CollisionHeader D_06001238; // Zora's Domain entrance block of ice collision
-extern Gfx D_06002490[];           // Water Temple entrance lock
+extern Gfx gSpot06ObjectsDL0[];           // Lake Hylia lowered water
+extern Gfx gSpot06ObjectsDL1[];           // Lake Hylia raised water
+extern Gfx gSpot06ObjectsDL2[];           // Water Temple entrance gate
+extern CollisionHeader gSpot06ObjectsCol0; // Water Temple entrance gate collision
+extern Gfx gSpot06ObjectsDL3[];           // Zora's Domain entrance block of ice
+extern CollisionHeader gSpot06ObjectsCol1; // Zora's Domain entrance block of ice collision
+extern Gfx gSpot06ObjectsDL4[];           // Water Temple entrance lock
```

Comment:
More usable documentation, although I've suggested names for these already anyway.